testsuite/features/build_validation/smoke_tests/smoke_tests.template

```diff
@@ -137,8 +137,36 @@ Feature: Smoke tests for <client>
     Then I should see a "1 package install has been scheduled for" text
     And I wait until event "Package Install/Upgrade scheduled by admin" is completed
 
+  Scenario: Test Prometheus exporter formula on the <client>
+    Given I am on the Systems overview page of this "<client>"
+    When I follow "Formulas" in the content area
+    And I check the "prometheus-exporters" formula
+    And I click on "Save"
+    Then I wait until I see "Formula saved" text
+    # Configure Prometheus exporter formula
+    When I follow "Prometheus Exporters" in the content area
+    And I click on "Expand All Sections"
+    And I should see a "Enable and configure Prometheus exporters for managed systems." text
+    And I check "node" exporter
+    And I check "apache" exporter
+    And I check "postgres" exporter
+    And I click on "Save"
+    Then I should see a "Formula saved" text
+    # Apply highstate for Prometheus exporters
+    When I follow "States" in the content area
+    And I click on "Apply Highstate"
+    Then I should see a "Applying the highstate has been scheduled." text
+    And I wait until event "Apply highstate scheduled by admin" is completed
+    # Visit monitoring endpoints on the minion
+    When I wait until "node" exporter service is active on "<client>"
+    And I visit "Prometheus node exporter" endpoint of this "<client>"
```

Comment:
these are tests that it works.

`Then the "Prometheus node exporter" of this "<client>" should answer`

Reply:
this wasn't added by me, it already existed, and all monitoring tests use this step :sweat_smile: should I rephrase this anyway?

Reply:
??? I can see it added in the steps file below

Reply:
but it wasn't, I just moved it. I didn't change anything in the exporter formulas test, just moved it up so I'd have the monitoring server tests all together